### PR TITLE
Don't allow every single spider to be player controllable

### LINKED ISSUE #15446: Any spider can be clicked on by ghosts to be controlled
Previously player controlled spiders created spiders that could be controlled by players (and it was pop up spam)

Now any spider can be clicked on at any time regardless of origin, leading to frequent spider apocalypses.

### PR DESCRIPTION
**Pull request by optimumtact**
Fixes #15446

### CODE REVIEW

File: code/game/objects/effects/spiders.dm [hunk 1]
```diff
@@ -202,6 +202,7 @@
 			S.poison_type = poison_type
 			S.faction = faction.Copy()
 			if(player_spiders)
+                S.playable_spider = TRUE
```
Comment (optimumtact) [inline code/game/objects/effects/spiders.dm]: the fuck

Comment (GinjaNinja32) [inline code/game/objects/effects/spiders.dm]: spaces vs tabs?

**Comment by Zilenan91** [comment]
Does this prevent players from clicking on and becoming the green eyed spiders or does it prevent players from ever becoming green eyed spiders period

**Comment by Buggy123** [comment]
:-1: muh fun removal

**Comment by optimumtact** [comment]
It returns spiders to the status quo of only being gettable through specific methods (sentience potion I think)

**Comment by Zilenan91** [comment]
So this only allows the purple eyed spiders to be clicked on or does this allow the red eyed ones as well?

**Comment by optimumtact** [comment]
YOU CAN ONLY CLICK ON SPIDERS THAT ARE PLAYER CONTROLLABLE I DON'T KNOW WHAT EYES HAVE TO DO WITH IT

**Comment by Dorsisdwarf** [comment]
This only allows PC spiders (IE: Admins set playercontrolled to 1, or sentience potion) to be clicked to control.

**Comment by Supermichael777** [comment]
The orange a voice of reason?
oh well :+1:

**Comment by octareenroon91** [comment]
Spiderlings grow up into playable spiders. Is there a chance of egg clusters forming in the away missions that seemed to have sparked this business?

@Zilenan91 Any type of spider can still become player-controlled.

**Comment by Shadowlight213** [comment]
What this does is restore the original thing of spiders can only become player controlled if the spider who laid the eggs was player controlled.

**Comment by optimumtact** [comment]
@Supermichael777 I accidentally introduced this bug in the first place so not completely reasonable

mean people forced me to do it.

File: code/modules/mob/living/simple_animal/hostile/giant_spider.dm [hunk 2]
```diff
@@ -48,18 +48,19 @@
...
-		if(istype(ghost))
+		if(istype(ghost) && playable_spider)
 			humanize_spider(ghost)
 
 /mob/living/simple_animal/hostile/poison/giant_spider/attack_ghost(mob/user)
 	humanize_spider(user)
 
 /mob/living/simple_animal/hostile/poison/giant_spider/proc/humanize_spider(mob/user)
-	if(key)//Someone is in it
+	if(key || !playable_spider)//Someone is in it
```
Comment (duncathan) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: Maybe update the comment

**Comment by octareenroon91** [comment]
Oh I see now @Shadowlight213 .  I skimmed through too quickly.

**Comment by GunHog** [comment]
Imagine if Changelings were still enabled...I know that if I were one, I would be mass producing spiders!

**Comment by YakumoChen** [comment]
fun removal :-1:

**Comment by Niknakflak** [comment]
:+1: This wouldn't be a problem if people didn't abuse it.  You lost your chance

**Comment by Steelpoint** [comment]
It's unfortunate this feature is being removed.

Also Niknak what the heck do you expect when everyone who is dead gets pop ups constantly about a new spider to take over?

If it's possible I think a rebalancing would be better, such as limiting the rate of player controllable spiders being offered up for control or making only a certain sub-section of spiders controllable.

**Comment by KorPhaeron** [comment]
@Steelpoint 

You entirely misunderstand the PR. When oranges fixed the pop-ups he accidentally made it so you can take over any spider at any time, even ones that were previously never offered to players to control.

**Comment by Steelpoint** [comment]
Ah.

Well I still think that whole scenario was very fun and exciting, but a bug's a bug.

**Comment by Ultimate-Chimera** [comment]
Spiders too OP im ded. :+1:

**Comment by optimumtact** [comment]
Yeah lets go with accidentally

**Comment by Shadowlight213** [comment]
The number of comments thinking this is removing a feature is astounding.

**Comment by InsaneHyena** [comment]
A fun feature somehow bled into SS13 code, better remove it now.

**Pull request merged**